engine/tests/mechanics/games/test_credentials_scoring_config.py

```
@@ -0,0 +1,194 @@
+# --------------------------------------------------------------------------- #
+# a decree regime overrides the matrix
+# --------------------------------------------------------------------------- #
+class TestDecreeRegime:
+    """'Arrest everyone, any non-arrest is a failure.'"""
+
+    DECREE = {
+        "pass": {"pass": 0, "deny": 2, "arrest": 5},
+        "deny": {"pass": 2, "deny": 0, "arrest": 5},
+        # under decree a non-arrest is a hard failure, not a mild hedge
+        "arrest": {"pass": 5, "deny": 5, "arrest": 0},
+    }
```

Ruff (0.15.15)

[warning] Mutable default value for class attribute

(RUF012)